package.json

```diff
@@ -70,53 +70,53 @@
   },
   "homepage": "https://github.com/uPortal-Project/uportal-app-framework",
   "devDependencies": {
-    "@commitlint/cli": "^7.0.0",
-    "@commitlint/config-conventional": "^7.0.0",
-    "@commitlint/travis-cli": "^7.0.0",
-    "autoprefixer": "^9.0.0",
-    "concurrently": "^3.1.0",
-    "eslint": "^5.0.0",
+    "@commitlint/cli": "^7.2.1",
+    "@commitlint/config-conventional": "^7.1.2",
+    "@commitlint/travis-cli": "^7.2.1",
+    "autoprefixer": "^9.3.0",
+    "concurrently": "^4.0.1",
+    "eslint": "^5.7.0",
```

Comment:
Looks like [the upgraded `eslint` version is very concerned about indentation](https://travis-ci.org/uPortal-Project/uportal-app-framework/jobs/445369884#L686) issues [not flagged by the status quo `eslint`](https://travis-ci.org/uPortal-Project/uportal-app-framework/jobs/445353514#L742). If we merge this before addressing or silencing the indentation then the CI build is going to turn red.

Reply:
994ebfa has most of the lint fixes, changing the intent cause a few `max-len` warnings, that will need to be looked into.